tests/test_apply.py

```diff
@@ -633,26 +649,6 @@ def circuit(x):
 
         assert np.isclose(circuit(p), 1, atol=tol, rtol=0)
 
-    def test_nonzero_shots(self, tol):
```

Comment:
I removed this as LightningGPU doesn't currently support `shots > 0`.

Reply:
I think this is fine, but keeping it as an expected failure for now. When we get support for `shots>0` that test will let us know an update is needed.